privval/file.go

```diff
@@ -299,6 +309,18 @@ func (pv *FilePV) String() string {
 	)
 }
 
+func (pv *FilePV) signOracleVote(vote *oracleproto.GossipVote) error {
+	signBytes := types.OracleVoteSignBytes(vote)
+	sig, err := pv.Key.PrivKey.Sign(signBytes)
+	if err != nil {
+		return err
+	}
+	vote.SignedTimestamp = uint64(time.Now().Unix())
```

Comment:
i think this needs to be part of the signed bytes, otherwise gossipers can change the timestamp and screw things up